demo/hooks/useAgentManager.ts

```diff
@@ -23,9 +32,24 @@ export function useAgentManager(props: UseAgentManagerOptions) {
 
     const [isSpeaking, setIsSpeaking] = useState(false);
     const [messages, setMessages] = useState<Message[]>([]);
+    const [videoState, setVideoState] = useState(StreamingState.Stop);
+    const [agentState, setAgentState] = useState(AgentState.Idle);
     const [srcObject, setSrcObject] = useState<MediaStream | null>(null);
     const [agentManager, setAgentManager] = useState<AgentManager | null>(null);
     const [connectionState, setConnectionState] = useState<ConnectionState>(ConnectionState.New);
+    const streamType = agentManager?.getStreamType();
+
+    useEffect(() => {
```

Comment:
One of the simplest, most beautiful pieces of code we have in this repo. Good work.